utils/vara-ui/src/components/button/button.module.scss

```diff
@@ -1,46 +1,44 @@
-@keyframes blink {
-  0% {
-    background-position: -100% 0;
-  }
-
-  100% {
-    background-position: 100% 0;
-  }
-}
-
 .button {
-  border: none;
-  cursor: pointer;
-  padding: 0;
-  font-family: inherit;
-
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  gap: 10px;
+  gap: 8px;
 
-  font-weight: 700;
-  line-height: 18px;
-  letter-spacing: 0.03em;
+  padding: 0;
 
-  border-radius: 2px;
-  transition: all 0.25s;
+  font-family: inherit;
+  font-weight: 600;
+  line-height: 24px;
+
+  border: none;
+  border-radius: 4px;
+  transition-property: color, background-color, box-shadow;
+  transition-duration: 250ms;
+
+  cursor: pointer;
+  outline: none;
+  outline-offset: 2px;
 
   &:disabled {
-    pointer-events: none;
+    cursor: not-allowed;
```

Comment:
`cursor: not-allowed` on a disabled element with `cursor: pointer` is an UX overkill.